src/js/binaryen.js-post.js

```diff
@@ -2567,7 +2567,7 @@ function wrapModule(module, self = {}) {
   self['removeExport'] = function(externalName) {
     return preserveStack(() => Module['_BinaryenRemoveExport'](module, strToStack(externalName)));
   };
-  self['setMemory'] = function(initial, maximum, exportName, segments = [], shared = false, memory64 = false, internalName) {
+  self['setMemory'] = function(initial, maximum, exportName, segments = [], shared = false, memory64 = false, internalName = 0) {
```

Comment:
Isn't this should be `internalName = null` or `internalName = "0"`?

Reply:
I think 0 or null work, as the call to `strToStack` will treat them as "nothing". "0" however would be used as a name, which is not good.

Maybe null or undefined are best. `undefined` is the old value actually, so that might be best.

Reply:
`null` seems clearest to me, since we really are trying to get a null ptr as the default here.